css-layout-classes/styles.css

```diff
@@ -0,0 +1,35 @@
+#main-container {
+  padding: 0 100px;
+  font-family:
+    Arial,
+    Helvetica,
+    sans-serif;
+  font-size: large;
+}
+
+.main-head {
+  font-size: xx-large;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
```

Comment:
you typically want to avoid using more that just `display:flex` for the row class since its primary job is just to be a flex container. Also, you don't need flex-wrap or justify-content for this exercise since the columns in each row will take up 100% of the width.